docs/source/general/freeradius.rst

```diff
@@ -64,8 +64,11 @@ Now enable the ``sql`` and ``rest`` modules:
 .. code-block:: shell
 
     cd mods-enabled/
-    ln -s /etc/freeradius/mods-available/sql /etc/freeradius/mods-enabled/sql
-    ln -s /etc/freeradius/mods-available/rest /etc/freeradius/mods-enabled/rest
+    ln -s /etc/freeradius/3.0/mods-available/sql /etc/freeradius/3.0/mods-enabled/sql
+    ln -s /etc/freeradius/3.0/mods-available/rest /etc/freeradius/3.0/mods-enabled/rest
+    
+.. note::
+    Where *3.0* is version of freeradius. In the future there may be another version, not 3.0
```

Comment:
Could you also add a note that says that on some distributions the 3.0/ dir is missing and the path is `/etc/freeradius/mods-available` please?

Reply:
Ok, I will use virtualenv (just I have a lot of VMs special for this task). I will try to use and documentate virtualenv.
Currently Im on *Ubuntu 17.04*, but I will check it on `Ubuntu 17.10`, `Debian 8`, `Debian 9` (ofc previously update distro)

Reply:
no worries, you don't need to check, I know for sure because I have some servers running Ubuntu 16.10 and other older versions of debian

Reply:
please add the note I asked in the first command